integration/tools/validation/src/main/java/alluxio/cli/NativeLibValidationTask.java

```diff
@@ -21,7 +19,7 @@
  * Validates the Java native libs defined in the system property.
  * */
 @ApplicableUfsType(ApplicableUfsType.Type.ALL)
-public class NativeLibValidationTask extends AbstractValidationTask {
+public class NativeLibValidationTask implements ValidationTask {
```

Comment:
`AbstractValidationTask` is now gone, as the only method it implements does not exist anymore.